ui/src/components/DownloadListItem/index.js

```diff
@@ -2,31 +2,57 @@ import React from 'react';
 import { withPolling } from 'ui/utils/hocs';
 import { compose, withProps } from 'recompose';
 import OptionListItem from 'ui/components/OptionListItem';
+import { activeOrganisationSettingsSelector } from 'ui/redux/modules/auth';
+import { connect } from 'react-redux';
+import moment from 'moment';
+import createdAt from 'ui/containers/Owner/CreatedAt';
 
-const DownloadListItem = ({ model, deleteModel }) => {
-  if (model.get('isReady')) {
+const DownloadListItem = (props) => {
+  // TODO: remove exporationExprots
+  const { download, deleteDownloadModel } = props;
+
+  let expireExportsString = '';
+
+  if (download.get('expireTTL')) {
+    const expireExports = moment(download.get('expireTTL'));
+    const formattedExpireExports = createdAt(expireExports, {
```

Comment:
We shouldn't be using the `createdAt` function here since it makes no sense semantically when you read it. Duplicate the `createdAt` function and make an `expiresAt` function.

Reply:
I agree with @ryansmith94 

I included this formatting into function `getDownloadDisplayName` (added at https://github.com/LearningLocker/learninglocker/commit/80fb50f846b2ce69ca86bdcbe27948c62276945d)

Reply:
Nice Hota, much prefer that.